src/SFML/Window/WindowBase.cpp

```diff
@@ -70,6 +70,14 @@ WindowBase::~WindowBase()
 }
 
 
+////////////////////////////////////////////////////////////
+WindowBase::WindowBase(WindowBase&&) noexcept = default;
```

Comment:
Wouldn't you need to address `WindowsBaseImpl::fullscreenWindow`?
If you move an object that this global variable currently points to, the new object would not take its place.